sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/model/Person.java

```diff
@@ -75,10 +75,10 @@ public Person decode(InputStream inStream)
   public final String name;
 
   @JsonProperty
-  private final String emailAddress;
+  public final String emailAddress;
 
   @JsonProperty
-  private final String creditCard;
+  public final String creditCard;
```

Comment:
<!--new_thread; commit:84f538f392fa0a3244e7ffe4819adbef86d6e289; resolved:1-->
Something about this is funny ;-)

Reply:
:)